src/components/DropdownBase/DropdownBase.js

```diff
@@ -50,11 +53,39 @@ export default class DropdownBase extends React.Component {
   }
 
   render() {
+    const info = this.props.info ? (
+      <div className={LABEL_CLASSES.HELP}>
+        <Icon
+          uxpId={`infoIcon-${this.state.id}`}
+          icon="circle-info-outline"
+          size="xs"
+          color="primary"
+          mode="button"
+          popover
+          popoverTitle={this.props.infoPopoverTitle}
+          popoverDescription={this.props.infoPopoverDescription}
+          popoverPosition={this.props.infoPopoverPosition}
+        />
+      </div>
+    ) : null;
+
+    const label = this.props.label ? (
+      <Label
```

Comment:
We have seen that the `Label` component already has the property **info** and can be included through this component, can we reuse the Label info prop?

Reply:
Good catch, yes we can. Will update it.